src/pages/instances/forms/InstanceFormMenu.tsx

```diff
@@ -87,7 +88,9 @@ const InstanceFormMenu: FC<Props> = ({
               />
               <MenuItem label={GPU_DEVICES} {...menuItemProps} />
               <MenuItem label={PROXY_DEVICES} {...menuItemProps} />
-              <MenuItem label={OTHER_DEVICES} {...menuItemProps} />
+              {useSupportedFeatures().hasMetadataConfiguration ? (
+                <MenuItem label={OTHER_DEVICES} {...menuItemProps} />
+              ) : null}
```

Comment:
A bit easier to read I think
```suggestion
              {hasMetadataConfiguration && <MenuItem label={OTHER_DEVICES} {...menuItemProps} />}
```